### PR TITLE
add parent Id input to admin v3

### PR DESCRIPTION
**Pull request by denisdimitrov**
- Adds parent id field to admin v3
- Remove styled components from Advanced settings screen
- Remove unnecessary components and replace them with InputField or Toggler
- Some CSS changes

### CODE REVIEW

File: src/pages/AdminAdvancedSettings/AdminAdvancedSettings.tsx
Comment (dmitrymarchuk) [inline src/pages/AdminAdvancedSettings/AdminAdvancedSettings.tsx:81]: Why after merge?

**Review by dmitrymarchuk — APPROVED** [review]
Great refactoring here!

File: src/pages/AdminAdvancedSettings/AdminAdvancedSettings.tsx
Comment (denisdimitrov) [reply src/pages/AdminAdvancedSettings/AdminAdvancedSettings.tsx:81]: What I mean by that is after the PR for NotFound component is merged then we need to add it here. Or just in that PR. https://github.com/sparkletown/sparkle/pull/2219

File: src/pages/AdminAdvancedSettings/AdminAdvancedSettings.tsx
Comment (goran-peoski-work) [inline src/pages/AdminAdvancedSettings/AdminAdvancedSettings.tsx:103]: 🎉

**Review by goran-peoski-work — CHANGES_REQUESTED** [review]
Few comments about the benefits of the type rename and a suggestion of reworking Toggler in a different manner.

File: src/components/atoms/Toggler/Toggler.tsx
Comment (denisdimitrov) [reply src/components/atoms/Toggler/Toggler.tsx:10]: I'll just remove it and if anyone needs it will pass it in the proper way.

File: src/pages/Admin/AdvancedSettings/AdvancedSettings.scss
Comment (denisdimitrov) [reply src/pages/Admin/AdvancedSettings/AdvancedSettings.scss]: Fixed in: https://github.com/sparkletown/sparkle/pull/2340/commits/6aa45c1c0dc93f7258b7e1541f2e1db2d1a3c541

File: src/components/atoms/Toggler/Toggler.tsx
Comment (denisdimitrov) [reply src/components/atoms/Toggler/Toggler.tsx]: Fixed in: https://github.com/sparkletown/sparkle/pull/2340/commits/6aa45c1c0dc93f7258b7e1541f2e1db2d1a3c541

File: src/pages/Admin/AdvancedSettings/AdvancedSettings.tsx
Comment (goran-peoski-work) [inline src/pages/Admin/AdvancedSettings/AdvancedSettings.tsx:57]: Can I ask a minor adjustment here so we actually capture possible errors, even if just in console?

```tsx
    updateVenue_v2(
      {
        name: venue.name,
        worldId: venue.worldId,
        ...data,
      },
      user
    ).catch((e) => console.error(AdvancedSettings.name, e));
```

**Review by goran-peoski-work — CHANGES_REQUESTED** [review]
Got unrelated field error while trying to update just the parent id
sparkle-grid-columns-01

File: src/pages/Admin/AdvancedSettings/AdvancedSettings.tsx
Comment (denisdimitrov) [reply src/pages/Admin/AdvancedSettings/AdvancedSettings.tsx:57]: The columns are now fixed and console.error has been added.

Comment (goran-peoski-work) [reply src/pages/Admin/AdvancedSettings/AdvancedSettings.tsx:57]: @denisdimitrov 
> How about Bugsnag?
Nothing against it. As long as errors aren't ignored. Any particular context we want to add to Admin related errors?

File: functions/venue.js
Comment (goran-peoski-work) [inline functions/venue.js:382]: No check for self reference on input causes bug.
sparkle-admin-parent-id-01

File: src/pages/Admin/AdvancedSettings/AdvancedSettings.tsx
Comment (goran-peoski-work) [inline src/pages/Admin/AdvancedSettings/AdvancedSettings.tsx:74]: Can use components like

- https://github.com/sparkletown/sparkle/tree/staging/src/components/organisms/AdminVenueView/components/AdminPanel
- https://github.com/sparkletown/sparkle/tree/staging/src/components/organisms/AdminVenueView/components/AdminSidebar
- https://github.com/sparkletown/sparkle/tree/staging/src/components/organisms/AdminVenueView/components/AdminSidebarTitle

to unify admin UI/UX. Especially useful to not worry about proper semantic use of `h1`, `h2` etc.

**Review by goran-peoski-work — CHANGES_REQUESTED** [review]
One bug. Looking good otherwise.

**Review by goran-peoski-work — APPROVED** [review]
Approving with the hope a fix for the circular `parentId` is made through additional PR sooner than later.

**Review by goran-peoski-work — APPROVED** [review]
LGTM. 

Some nitpicks and a request all user facing instances of Room/Rooms/room/rooms be dealt with like in https://github.com/sparkletown/sparkle/pull/2339/files#diff-25d09eff7b10c21e2970505b6703bf98d591a6d46ff301d7a6b4166fb1136cfb

**Pull request merged**
